### ISSUE TITLE
Closing cursor with changes feed

### ISSUE THREAD

**Issue by Wildsoft** [AtnNn/haskell-rethinkdb#71]
No sure if it is really an issue , just wanted to confirm what is the right way to release Cursor obtained to receive change feeds (changes). There is no close method on Cursor (https://www.rethinkdb.com/api/javascript/close-cursor/) so for now I just spawn thread in which I invoke each function. When I no longer need changesfeed I cancel the thread. My concern is that it would cause resource leak if cursor is not released correctly at that point. Thanks.

**Comment by codedmart** [comment]
@Wildsoft Are you looking for this [close](https://github.com/AtnNn/haskell-rethinkdb/blob/c7df4f219f1e7251629a9e90280f1ebdfed9ec8c/Database/RethinkDB/Network.hs#L382-L387)?

**Comment by Wildsoft** [comment]
@codedmart No, I was not looking to close connection. I wanted to close cursor and release associated resources. Like: https://www.rethinkdb.com/api/javascript/close-cursor/
